### PR TITLE
chore: Updated Attio deep connector content in the provider guides

### PR DESCRIPTION
**Pull request by sanjay-a01**

### CODE REVIEW

File: src/provider-guides/attio.mdx [hunk 1]
```diff
@@ -6,8 +6,30 @@ title: "Attio"
...
+### Supported Objects
+The Attio connector supports reading from the following objects:
+- [Object](https://developers.attio.com/reference/get_v2-objects)
+- [Lists](https://developers.attio.com/reference/get_v2-lists)
+- [WorkspaceMembers](https://developers.attio.com/reference/get_v2-workspace-members)
+- [Notes](https://developers.attio.com/reference/get_v2-notes)
+- [Tasks](https://developers.attio.com/reference/get_v2-tasks)
+- [Webhooks](https://developers.attio.com/reference/get_v2-webhooks)
+
+The Attio connector supports writing from the following objects:
```
Comment (RajatPawar) [inline src/provider-guides/attio.mdx:22]:
```suggestion
The Attio connector supports writing to the following objects:
```

File: src/provider-guides/attio.mdx [hunk 2]
```diff
@@ -65,3 +87,17 @@ Follow the steps below to create an Attio app and add the Ampersand redirect URL
    ![Ampersand Integration](/images/provider-guides/07458e9-attio2.gif)
 
 6. Click **Save Changes**.
+
+## Using the connector
+
+This connector uses API Key auth, which means that you do not need to set up a Provider App before getting started. (Provider apps are only required for providers that use OAuth2 Authorization Code grant type.)
```
Comment (RajatPawar) [inline src/provider-guides/attio.mdx:93]:
```suggestion
This connector uses API Key auth, which means that you do not need to set up a Provider App before getting started. Provider apps are only required for providers that use OAuth2 (Authorization Code) grant type.
```

File: src/provider-guides/attio.mdx [hunk 3]
```diff
@@ -65,3 +87,17 @@ Follow the steps below to create an Attio app and add the Ampersand redirect URL
...
+
+To start integrating with Attio:
+- Create a manifest file like the [example](https://github.com/amp-labs/samples/blob/main/attio/amp.yaml), see [Define Integrations](/define-integrations/overview) for more info.
```
Comment (RajatPawar) [inline src/provider-guides/attio.mdx:96]:
```suggestion
- Create a manifest file using the [example](https://github.com/amp-labs/samples/blob/main/attio/amp.yaml), see [Define Integrations](/define-integrations/overview) for more info.
```

**Review by RajatPawar — APPROVED** [review]
Approving, please address the comments before merging it in

**Review by laurenzlong — APPROVED** [review]

**Pull request merged**